homeassistant/components/knx/light.py

```diff
@@ -60,16 +90,36 @@ def async_add_entities_discovery(hass, discovery_info, async_add_entities):
 def async_add_entities_config(hass, config, async_add_entities):
     """Set up light for KNX platform configured within platform."""
     import xknx
+
+    group_address_tunable_white = None
+    group_address_tunable_white_state = None
+    group_address_color_temp = None
+    group_address_color_temp_state = None
+    if config[CONF_COLOR_TEMP_MODE] == ColorTempModes.absolute:
+        group_address_color_temp = config.get(CONF_COLOR_TEMP_ADDRESS)
+        group_address_color_temp_state = \
+            config.get(CONF_COLOR_TEMP_STATE_ADDRESS)
+    elif config[CONF_COLOR_TEMP_MODE] == ColorTempModes.relative:
+        group_address_tunable_white = config.get(CONF_COLOR_TEMP_ADDRESS)
+        group_address_tunable_white_state = \
+            config.get(CONF_COLOR_TEMP_STATE_ADDRESS)
+
     light = xknx.devices.Light(
         hass.data[DATA_KNX].xknx,
-        name=config.get(CONF_NAME),
+        name=config[CONF_NAME],
         group_address_switch=config.get(CONF_ADDRESS),
```

Comment:
`CONF_ADDRESS` is required too. See line 52
```suggestion
        group_address_switch=config[CONF_ADDRESS],
```